_drgn.pyi

```diff
@@ -495,6 +498,19 @@ class Register:
 host_platform: Platform
 """The platform of the host which is running drgn."""
 
+class Language:
+    """
+    A ``Language`` represents a programming language supported by drgn.
+
+    This class cannot be constructed; there are singletons for the supported
+    languages.
+    """
+    name: str
+    """Name of the programming language."""
+
+    C: "Language"
```

Comment:
Sorry for the (continued) issues, but this dosen't seem to be propogated through properly (the docstring at least).

From the shell:

```
class Language(builtins.object)
 |  A ``Language`` represents a programming language supported by drgn.
 |  
 |  This class cannot be constructed; there are singletons for the supported
 |  languages.
 |  
 |  Methods defined here:
 |  
 |  __repr__(self, /)
 |      Return repr(self).
 |  
 |  ----------------------------------------------------------------------
 |  Data descriptors defined here:
 |  
 |  name
 |      str
 |      
 |      Name of the programming language.
 |  
 |  ----------------------------------------------------------------------
 |  Data and other attributes defined here:
 |  
 |  C = Language.C
```

Any idea why this might be? If not, I can try to triage how the help text is propogated tomorrow :)